### PR TITLE
Attributed strings can be used as picker row titles.

### PR DESCRIPTION
**Pull request by xjki**
This pull request allows to provide attributed strings (eg text with icon) as picker rows.
Also fixed documentation warnings and accessibility for picker title.

### CODE REVIEW

File: Pickers/ActionSheetMultipleStringPicker.h [hunk 1]
```diff
@@ -35,28 +35,30 @@ typedef void(^ActionMultipleStringDoneBlock)(ActionSheetMultipleStringPicker *pi
...
  *  Create and display an action sheet picker.
  *
  *  @param title             Title label for picker
  *  @param data              is an array of strings to use for the picker's available selection choices
- *  @param index             is used to establish the initially selected row;
- *  @param target            must not be empty.  It should respond to "onSuccess" actions.
- *  @param successAction
+ *  @param indexes           is used to establish the initially selected rows;
+ *  @param target            must not be empty. It should respond to "onSuccess" actions.
+ *  @param successAction     successActin
```
Comment (skywinder) [inline Pickers/ActionSheetMultipleStringPicker.h]: Please fix this typo 👍

File: Pickers/ActionSheetStringPicker.m [hunk 2]
```diff
@@ -168,19 +180,8 @@ - (UIView *)pickerView:(UIPickerView *)pickerView viewForRow:(NSInteger)row forC
...
-    // otherwise, use the description, just like the toString() method in Java
-    // else, use String with no text to ensure this delegate do not return a nil value.
-    
-    if ([obj isKindOfClass:[NSString class]])
-        attributeTitle = [[NSAttributedString alloc] initWithString:obj attributes:self.pickerTextAttributes];
-    
-    if ([obj respondsToSelector:@selector(description)])
-        attributeTitle = [[NSAttributedString alloc] initWithString:[obj performSelector:@selector(description)] attributes:self.pickerTextAttributes];
     
+    NSAttributedString *attributeTitle = [self pickerView:pickerView attributedTitleForRow:row forComponent:component];
```
Comment (skywinder) [inline Pickers/ActionSheetStringPicker.m]: 👍  great. seems it should works! Please fix typo above and I will happy to merge!

**Comment by xjki** [comment]
Please delete this request. Created another pull request https://github.com/skywinder/ActionSheetPicker-3.0/pull/403

File: .travis.yml [hunk 3]
```diff
@@ -8,14 +8,13 @@ script:
...
-
 # xcode_workspace: ActionSheetPicker-3.0.xcworkspace
 # xcode_scheme: ActionSheetPickerTests
 # xcode_sdk: iphonesimulator
 
 notifications:
   email:
     recipients:
-      - sky4winder+actionSheetPicker@gmail.com
+      - jki@jki.lv
```
Comment (skywinder) [inline .travis.yml]: 🧐 Travis is not worked correctly yet. If you want to setup it - please make a list of recipients

File: Pickers/SWActionSheet.m [hunk 4]
```diff
@@ -208,13 +208,6 @@ - (BOOL)prefersStatusBarHidden {
 	return [UIApplication sharedApplication].statusBarHidden;
 }
 
-- (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interfaceOrientation
```
Comment (skywinder) [inline Pickers/SWActionSheet.m]: Are you sure, that we can delete this check? I added it to fix something (shame of me, didn't put comment there).. But still.

**Comment by skywinder** [comment]
> Please delete this request. Created another pull request #403

oh, missed it. I will close it in favor of #403

**Pull request closed without merging**